filechooser-portal/Main.vala

```diff
@@ -178,11 +170,17 @@ public class Files.FileChooserPortal : Object {
             dialog.set_current_folder (FileUtils.sanitize_path (options["current_folder"].get_bytestring ()));
         }
 
+        var supplied_uri = "";
         if ("current_file" in options) {
-            dialog.set_uri (FileUtils.sanitize_path (
-                options["current_file"].get_bytestring (),
-                Environment.get_home_dir ()
-            ));
+            supplied_uri = FileUtils.sanitize_path (
+                options["current_file"].get_bytestring (), Environment.get_home_dir ()
+            );
+
+            if (supplied_uri != null) {
+                dialog.set_uri (supplied_uri);
+            } else {
+                supplied_uri = "";
```

Comment:
Is this supposed to be `dialog.set_uri ("")`? This line does nothing, setting `supplied_uri` to the value it already has.

Reply:
This was to ensure `supplied_uri` is not `null` but you are right, the `sanitize_path` function already does this.